packages/toolkit/src/view/knowledge/components/tabs/CatalogFilesTab.tsx

```diff
@@ -108,35 +103,31 @@ export const CatalogFilesTab = ({
     setIsFileDetailsOpen(false);
   };
 
-
   const handleDelete = (fileUid: string) => {
```

Comment:
@thewbuk Before delete it, we should first check whether the query state is success

if (!file.isSuccess){
  return
}